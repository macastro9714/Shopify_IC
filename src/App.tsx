import './styles/index.css';

import LandingPage from '@components/LandingPage';
import { history, store } from '@store';
import { ConnectedRouter } from 'connected-react-router';
import React from 'react';
import { Provider } from 'react-redux';
import { Route, Switch } from 'react-router-dom';

const App: React.FC = () => (
    <Provider store={store}>
        <ConnectedRouter history={history}>
            <Switch>
                <Route path="/" component={LandingPage} />
            </Switch>
        </ConnectedRouter>
    </Provider>
);

export default App;
