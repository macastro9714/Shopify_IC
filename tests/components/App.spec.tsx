import { shallow } from 'enzyme';
import React from 'react';

import App from '../../src/App';

describe('App component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<App />);

        expect(wrapper).toMatchSnapshot();
        wrapper.unmount();
    });
});
