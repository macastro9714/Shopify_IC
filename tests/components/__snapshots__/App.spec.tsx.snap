// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly 1`] = `
<h1>
  APP!
</h1>
`;
